src/diffusers/loaders/lora.py

```diff
@@ -986,6 +986,9 @@ def process_weights(adapter_names, weights):
                 raise ValueError(
                     f"Length of adapter names {len(adapter_names)} is not equal to the length of the weights {len(weights)}"
                 )
+
+            weights = [{"text_model": w} if w is not None else {"text_model": 1.0} for w in weights]
```

Comment:
Technically, `w` cannot be `None` here, right? This would require the weights to be a list with None elements.